packages/meditrak-app/app/home/HomeScreen.js

```diff
@@ -6,20 +6,15 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { Animated, StyleSheet, Text, FlatList, View } from 'react-native';
 
-import {
-  THEME_COLOR_ONE,
-  THEME_TEXT_COLOR_FOUR,
-  DEFAULT_PADDING,
-  getGreyShade,
-} from '../globalStyles';
+import { THEME_TEXT_COLOR_FOUR, DEFAULT_PADDING, getGreyShade } from '../globalStyles';
 import { FeedItemContainer } from '../social';
 import { HomeHeaderContainer } from './HomeHeaderContainer';
-import { TupaiaHeaderLogo } from '../widgets';
+import { TupaiaLogo } from '../widgets';
 import { HomeToolbarContainer } from './HomeToolbarContainer';
 
 class HomeScreenComponent extends PureComponent {
   static navigationOptions = {
-    headerTitle: () => <TupaiaHeaderLogo />,
+    headerTitle: () => <TupaiaLogo />,
```

Comment:
There's a default padding in TupaiaHeaderLogo that caused the logo not aligned with other components (screenshots: beyondessential/tupaia-backlog#1696 (comment)). So I removed the whole TupaiaHeaderLogo component because it only contained TupaiaLogo and that extra padding